src/App.css

```diff
@@ -1,38 +1,119 @@
-.App {
+:root {
```

Comment:
- [ ] All of the images are not showing up.  You could use example images found on the internet (link to them, so you don't have to use active storage if you don't want to).  

## Tips and extra help

_Steps:_

### In the backend

- Update the models to include the image_url field.
- Add the proper image_url fields to the tables in the database via migrations.
- Update controllers so the params and actions include the image_url, that way, your API will render it in your JSON.

### In the frontend

- Include the images on your JSON fetches to the API.
- Update the redux action creators and reducers.
- Update the JSX in the proper components to include the images.
- Profit.


That way, your project will look beautiful, and professional. 💪